newIDE/app/src/PlatformSpecificAssetsEditor/PlatformSpecificAssetsDialog.js

```diff
@@ -227,6 +246,13 @@ export default class PlatformSpecificAssetsDialog extends React.Component<
         .getPlatformSpecificAssets()
         .set('android', `icon-${size}`, androidIconResourceNames[index]);
     });
+    project
+      .getPlatformSpecificAssets()
```

Comment:
same here, do we need to call it every time?

Reply:
Fixed